viewsgetter.bat

```diff
@@ -12,14 +12,15 @@ REM this gets all the data that has to do with the user's repos, but we dont rea
 curl "https://api.github.com/users/%user%/repos">%cd%\Repos.txt
 
 REM repo finder extracts the repo names so that we can do something with it
-javac -d out src/com/github/jojo2357/githubviewslogger/*.java
-java -cp $PWD/out/ com.github.jojo2357.githubviewslogger.RepoRefiner
+if not exist "out" mkdir out
```

Comment:
if not exist "ParsedData\Views" mkdir ParsedData\Views
if not exist "ParsedData\Clones" mkdir ParsedData\Clones